test/recompilation-tests/recompiler-off/check.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+set -e
+
+PREFIX=$(dirname "$0")
+
+echo "Testing with deleted artifact on recompiler option set to false"
+rm -rf starknet-artifacts/contracts/contract.cairo/
+npx hardhat run --no-compile scripts/deploy.ts 2>&1 \
+    | head -n +4 \
+    | tail -1 \
+    | diff - "$PREFIX/artifact-expected.txt"
```

Comment:
You can use scripts/assert-contains.py (see how it's used in configuration-tests/with-disable-hint-compilation-option/check.sh)

Reply:
https://github.com/Shard-Labs/starknet-hardhat-plugin/pull/106/commits/4a768f95a2d4803c2ca231c024c5ebfa2df42c6e